docs/mkdocs/docs/tutorials/query_stats.md

```diff
@@ -0,0 +1,138 @@
+# Query Statistics
+
+> **Warning**: The Query Statistics API is unstable and not governed by ArcticDB's semantic versioning. It may change or be removed in future versions without notice.
+
+ArcticDB provides a Query Statistics API that allows you to collect and analyze performance metrics for operations performed on your data stores.
+This can be useful for debugging, performance optimization.
+The Query Statistics feature uses a global container to store all measurements. This means statistics from all Python threads in your application will be collected in the same structure. 
```

Comment:
I think it would be fine to move this to a "caveats" section instead of leading with it in the introduction